docker/entrypoint-ui-prod.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
Can you clean up this file a bit? Remove duplicate empty lines.

Also, please write a small documentation (as comment), what this file does... and more importantly, **why** it does that. I.e. explaining that this is workaround to get environment variables loaded.